hackathon_site/event/api_urls.py

```diff
@@ -7,4 +7,5 @@
 urlpatterns = [
     path("users/user/", api_views.CurrentUserAPIView.as_view(), name="current-user"),
     path("teams/team/", views.CurrentTeamAPIView.as_view(), name="current-team"),
+    path("event/leave_team/", views.LeaveTeamView.as_view(), name="leave-team"),
```

Comment:
This actually makes the path `/api/event/event/leave_team/`.

How about we make it `/api/event/teams/leave/`, now that I'm realizing there's a teams namespace. So that would be `path("teams/leave/, ...)` here